api/swap/_utils.ts

```diff
@@ -160,10 +160,11 @@ export async function handleBaseSwapQueryParams(
   const amountType = tradeType as AmountType;
   const amount = BigNumber.from(_amount);
 
-  const slippageToleranceNum = parseFloat(slippageTolerance);
-  const appFeePercentNum = appFeePercent
-    ? parseFloat(appFeePercent)
+  const slippageToleranceNum = slippageTolerance
+    ? parseFloat(slippageTolerance)
     : undefined;
+  const slippageNum = parseFloat(slippage);
+  const appFeePercentNum = appFee ? parseFloat(appFee) : undefined;
```

Comment:
Nit for consistency.
```suggestion
  const appFeeNum = appFee ? parseFloat(appFee) : undefined;
```